docs/getting-started.md

```diff
@@ -0,0 +1,333 @@
+# Getting started with containerd
+
+There are many different ways to use containerd.
+If you are a developer working on containerd you can use the `ctr` tool to quickly test features and functionality without writing extra code.
+However, if you are wanting to integrate containerd into your project we have an easy to use client package that allows you to work with containerd.
+
+In this guide we will pull and run a redis server with containerd using the client.
+We will assume that you are running a modern linux host for this example with a current build of `runc`.
+
+## Starting containerd
+
+You can download one of the lastest builds for containerd on the github releases page and then use your favorite process supervisor to get the daemon started.
+If you are using systemd, we have a `containerd.service` file in the root of the repository that you can use.
```

Comment:
`s/in the root/at the root/` (it's really just a nit and a personal preference though)